docs/geps/gep-01.rst

```diff
@@ -0,0 +1,148 @@
+3. output variables
+
+A convention on variable names should make the code easier to read and to maintain. In addition, it should lower the requirement to read the documentation (once there is one).
+
+General Guidelines
+-----------------
+- in principal, use lower case letters only. Exceptions are specified below.
+- longer names might be interrupted with underscores, e.g. `alg2_anzurechnendes_ek`
+- use of booelan type for dummy variables rather than integer. Boolean variables can be negated easily (mind the difference between `~` and `not`) and make the code easier to read.
+
+Input Variables
+-----------------
+
+First letter specifies type of variable
+----------------------------------------
+In the spirit of the Euromod_ framework, I suggest that the type of a variable is indicated by its first letter
+
+
++--------------+-----------------------------------+----------------------------------------------------------+
+| First letter | Type of Variable                  | Examples                                                 |
++==============+===================================+==========================================================+
+| d            | demographic                       | dage, dchild, dnchildren, dsex, deast, dmarried, dhhsize |
++--------------+-----------------------------------+----------------------------------------------------------+
+| y            | market income                     | ywage, yself, ycap, yrent                                |
++--------------+-----------------------------------+----------------------------------------------------------+
+| l            | labor market                      |  lhrs, lstatus                                           |
++--------------+-----------------------------------+----------------------------------------------------------+
+| b            | benefits (i.e. non-market income) | balim, boldagepen, bsocass                               |
++--------------+-----------------------------------+----------------------------------------------------------+
+| h            | housing                           | hsize, hrent, hheat                                      |
++--------------+-----------------------------------+----------------------------------------------------------+
+| x            | expenses                          | xchildcare                                               |
++--------------+-----------------------------------+----------------------------------------------------------+
+| a            | assets/wealth                     |  afinancial, aproperty                                   |
++--------------+-----------------------------------+----------------------------------------------------------+
+
+The table above demonstrates the virtue of the first letter identification. `hsize` can easily be misunderstood as the household size, which is a demographic variable. The first letter `h` avoids this misinterpretation.
```

Comment:
This.

In my world, the variable would be called `house_square_meters` or `house_n_rooms`, whatever is relevant.

Reply:
Or, `h_size_sqm` ;)